docker/demo/config/test-suite/target.avsc

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Yes, I tried to create a symlink for the schemas used across many places. The symlink works in intellij but fails during maven packaging and on travis since it doesn't pick up the symlink for some reason, had to duplicate this file.